test/services/cluster_stats/update_epoch_duration_service_test.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require "test_helper"
+
+module ClusterStats
+  class UpdateEpochDurationServiceTest < ActiveSupport::TestCase
+    setup do
+      @network = "testnet"
+
+      create(:epoch_wall_clock, epoch: 301, created_at: 16.days.ago)
+      create(:epoch_wall_clock, epoch: 311, created_at: 13.days.ago)
+      create(:epoch_wall_clock, epoch: 322, created_at: 10.days.ago)
+      create(:epoch_wall_clock, epoch: 333, created_at: 7.days.ago)
+      create(:epoch_wall_clock, epoch: 344, created_at: 4.days.ago)
+      create(:epoch_wall_clock, epoch: 355, created_at: 1.day.ago)
```

Comment:
Technically it's not an error, but why not use epochs numbers that actually represent epochs that happen one after another in time (whether 1, 2, 3, 4, 5 or 351, 352, 353, 354, 355, no difference). This is how it works in reality. Skipping numbers might be confusing for someone working on it in the future, for example might suggest that we want to skip some epochs, which it's not true.